tests/single-node/test.sh

```diff
@@ -103,3 +105,17 @@ EOF
         assert_fails 'make_shell kubectl get pv -o jsonpath={.items[*].status.phase} | grep Released > /dev/null' \
                 "PVs in Released state found"
 }
+
+test_prometheus_node_exporter_metrics() {
```

Comment:
This test rely on being on the server itself (yum and accessing prometheus cluster IP directly) but test are better than no test. As already discussed, we might want to move to a more feature-full framework (list of features to be defined)